ocaml/xapi/xapi_clustering.ml

```diff
@@ -146,17 +146,12 @@ let assert_cluster_host_has_no_attached_sr_which_requires_cluster_stack ~__conte
       Db.PBD.get_currently_attached ~__context ~self:pbd)
       (Db.Host.get_PBDs ~__context ~self:host) in
   let srs = List.map (fun pbd -> Db.PBD.get_SR ~__context ~self:pbd) pbds in
-  List.iter
+  if List.exists
     (fun sr ->
        let sr_sm_type = Db.SR.get_type ~__context ~self:sr in
-       (* XXX This check is a bit too conservative, because the SR requires
-          only one of these cluster stacks to be configured and running. *)
```

Comment:
I think we should keep this comment.

Reply:
The same comment (which no longer applies) or a similar comment pointing out the requirement of having at least one cluster stack configured and running?

Reply:
I think this comment still applies, and we should keep it exactly as it is.

Reply:
(This is about the semantics of `get_required_cluster_stacks` - what it means exactly if we get multiple. From an old comment I figured out that this means that we need at least one of them.)